src/package1/CustomArrayList.java

```diff
@@ -267,6 +270,13 @@ public List<String> subList(int fromIndex, int toIndex) {
      */
     @Override
     public String toString() {
-        return "";
+        StringBuilder s = new StringBuilder("[\"" + elements[0] + "\",");
+        for (int i = 1; i < size - 1; i++) {
+            if(elements[i] != null) {
+                s.append(" \"").append(elements[i]).append("\"").append(",");
+            }
+        }
+        s.append(" \"").append(elements[size - 1]).append("\"").append("]");
+        return s.substring(0);
```

Comment:
Why do you use s.substring(0)? Use return s.toString(); instead

Reply:
Method toString is not in the StringBuilder's documentation and I does not meet it. Thus used the methods that were in the documentation. I forgot about Object's methods. Corrected.